cmd/achcli/describe/file.go

```diff
@@ -162,6 +236,94 @@ func dumpAddenda99(w *tabwriter.Writer, a *ach.Addenda99) {
 	fmt.Fprintf(w, "      %s\t%s\t%s\t%s\t%s\t%s\n", a.ReturnCode, a.OriginalTrace, a.DateOfDeath, a.OriginalDFI, a.AddendaInformation, a.TraceNumber)
 }
 
+func dumpAddenda10(w *tabwriter.Writer, a *ach.Addenda10) {
+	if a == nil {
+		return
+	}
+
+	fmt.Fprintln(w, "\n      Addenda10")
```

Comment:
Should we remove these lines? Printing an IAT entry is already a ton of lines, so these seem to be unneeded.

Reply:
Sure thing. Do you want me to remove these for all addenda records?

Reply:
Yea

Reply:
@adamdecaf addressed your comment . please review whenever you have time